### ISSUE TITLE
Fossil Sample – attributes to display for search card

### ISSUE THREAD

**Issue by LeannePyle** [bcgov/BCHeritage#627]
Details Tab - moved to #669 

 
Search Result Card
- [x] Header: Repository or Institution number otherwise Field Number. Use string "No reference" if neither exists
- [x] Scientific Name(s)
- [x] Common Name(s)
- [x] Fossil Size Category
- [x] Age: Minimum Time – Maximum Time – include question marks if uncertainty boxes are checked
- [x] Stratigraphy: – Group, Formation or Informal name – only visible if data available

**Comment by bferguso** [comment]
@LeannePyle - can you please specify the format of the Stratigraphy part of the descriptor?
Eg: do we want it to be:
**Group**
\<Name of group\>
**Formation**
\<Name of formation\>
**Member**
\<Name of member\>
if this data exist, otherwise  

**Informal Map Unit or Name**
\<Informal Map Unit or Name\> if this data exists otherwise

**Other Informal / Other Unit Name**
\<Other Informal / Other Unit Name\>  if this data exists?

**Comment by LeannePyle** [comment]
that is correct - display the hierarchy Group-Formation-Member if exists, otherwise one of the other categories

**Comment by bferguso** [comment]
The primary descriptors function is behaving as required but there is an issue when re-indexing the database that is causing the scientific and common names to not be displayed (#746).

Moving to complete - other issue tracked in above issue.

**Comment by bferguso** [comment]
Automated cleanup
